fpga_interchange/arch.cc

```diff
@@ -895,6 +1011,28 @@ const std::vector<std::string> Arch::availablePlacers = {"sa",
 const std::string Arch::defaultRouter = "router2";
 const std::vector<std::string> Arch::availableRouters = {"router1", "router2"};
 
+bool parameter_compare(const std::string &database_param, const std::string &netlist_param)
+{
+
+    bool result = false;
+    try {
+        size_t idx;
+        int database_value = std::stoi(database_param, &idx, 0);
+        int netlist_value = std::stoi(netlist_param, &idx, 2);
```

Comment:
this looks a bit fragile to me, but I guess it's OK for a first go

Reply:
Exactly.  Parameter handling probably needs a lot of polish once the big fish are fried.